### ISSUE TITLE
Installer can't create shortcut

### ISSUE THREAD

**Issue by reddtoric** [DevelopersTree/permadelete#43]
Slight inconvenience; I had to look for permadelete to manually create the shortcut in _%appdata%/roaming_

Happens when creating shortcut for desktop and start menu.

During the installation:
Main window currently on:
```
Creating shortcuts...
C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Permadelete.Ink
```
Error window:
`|PersistFile::Save failed; code 0x80070005. Access is denied.`

OS: Windows 10 Pro N
Ver: 0.6

**Comment by luluhoc** [comment]
the same problem for me

**Comment by blitzkrieg99** [comment]
Proposed a fix for this issue in #55. Temporarily, just run the installer as administrator.

**Comment by mhmd-azeez** [comment]
Hello @blitzkrieg99's fix is merged and I published a new version here:
https://github.com/DevelopersTree/permadelete/releases/tag/v0.6.2
